src/main/webapp/pages/UserPage.jsp

```diff
@@ -92,16 +92,17 @@
             <br><br>
             <label for="steps" required>Steps:<br><i>Each step should be on its own line. No need to number them.</i></label><br>
             <textarea id="edit-steps" name="edit-steps" rows="4"></textarea>
-            <br><br>
-            <!-- Letting users publish their recipes is for after the MVP -->
-            <!-- <label for="privacy">Choose a privacy setting:</label>
-            <select id="privacy" name="privacy">
-            <option value="private">private</option>
-            <option value="public">public</option>
-            </select>
-            <br><br> -->
+            <div id="publishDiv">
+            <label for="privacy">Choose a privacy setting:</label>
+              <select id="privacy" name="privacy">
+                <option id="first"></option>
+                <option id="second"></option>
+              </select>
+              <p id="note"><i>Note: Once you publish a recipe you cannot edit the public version</i></p>
```

Comment:
Doesn't the public recipe get updated when the user edits the private recipe?

Reply:
oh yeah whoops